app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/ExpandedArticleFragment.java

```diff
@@ -0,0 +1,115 @@
+import edu.grinnell.grinnell_publications_android.R;
+
+
+/**
+ * Fragment that displays the expanded article content.
+ * Fragment allows for favoriting of the article.
+ */
+
+public class ExpandedArticleFragment extends Fragment implements UserInterface {
+
+    private ImageView mHeaderImage;
+    private CollapsingToolbarLayout mCollapsingToolbar;
+    private FloatingActionButton mFavoriteButton;
+    private TextView mArticleContent;
+    private Toolbar mArticleToolbar;
+
+
+    public ExpandedArticleFragment() {}
+
+    public static ExpandedArticleFragment newInstance() {
+        return new ExpandedArticleFragment();
+    }
+
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater,
+                             ViewGroup container,
+                             Bundle savedInstanceState) {
+        final View expandedArticleFragment = inflater.inflate(R.layout.fragment_extended_article, container, false);
+        mHeaderImage = (ImageView) expandedArticleFragment.findViewById(R.id.header_image);
+        mCollapsingToolbar = (CollapsingToolbarLayout) expandedArticleFragment.findViewById(R.id.collapsing_toolbar);
+        mFavoriteButton = (FloatingActionButton) expandedArticleFragment.findViewById(R.id.floating_action_button);
+        mArticleContent = (TextView) expandedArticleFragment.findViewById(R.id.article_content);
+        mArticleToolbar = (Toolbar) expandedArticleFragment.findViewById(R.id.article_toolbar);
+
+        initializeUI();
+        return expandedArticleFragment;
+    }
+
+    @Override
+    public void initializeUI(){
+        mArticleToolbar.setNavigationIcon(ContextCompat.getDrawable(getContext(), R.drawable.ic_action_back));
+        loadPlaceHolderData();
+        setOnClickListeners();
+
+    }
+
+
+    public void setOnClickListeners() {
+        mArticleToolbar.setNavigationOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                getFragmentManager().popBackStack();
+            }
+        });
+
+        mFavoriteButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Snackbar.make(v, "Sample article added to Favorites", Snackbar.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    /** Fills fragment with placeholder content */
+    private void loadPlaceHolderData(){
+        Drawable defaultImage = ContextCompat.getDrawable(getContext(), R.drawable.grinnell_gates);
+        setHeaderText("Sample title");
+        setContentText("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris feugiat diam sollicitudin est semper, eu porta libero pulvinar. Aenean at lacus rhoncus, pharetra elit ut, ultricies magna.");
```

Comment:
In as much as this is dummy text, please extract it into the `strings.xml` file and retrieve it via `ContextCompat.getResources(...)` so you can just change it there once it reflects here.